src/epub/content.opf

```diff
@@ -68,8 +68,8 @@
 		<meta property="group-position" refines="#collection-3">6</meta>
 		<dc:description id="description">A beautiful and virtuous young lady is entangled in the snares of a handsome but unscrupulous rake.</dc:description>
 		<meta id="long-description" property="se:long-description" refines="#description">
-			&lt;p&gt;&lt;i&gt;Clarissa Harlowe, or The History of a Young Lady&lt;/i&gt; is one of the longest novels in the English language. Written by &lt;a href="https://standardebooks.org/ebooks/samuel-richardson"&gt;Samuel Richardson&lt;/a&gt; over a period of several years and published in 1748, it is composed entirely of letters. Though this may seem daunting, the novel is highly regarded and is considered by many critics as one of the greatest works of English literature, appearing in several lists of the best British novels ever written.&lt;/p&gt;
-			&lt;p&gt;The novel tells the story of young Clarissa, eighteen years of age at the start of the novel. She is generally regarded by her family, neighbors, and friends as the most virtuous and kind young woman they know. But she is drawn into correspondence with Richard Lovelace, a well-born, rich young man regarded as something of a rake, when she attempts to reconcile a dispute between Lovelace and her rash brother. Lovelace, imagining this indicates her love for him, carries out a series of strategems which result in him essentially abducting her from her family, from whom Clarissa then becomes estranged.&lt;/p&gt;
+			&lt;p&gt;&lt;i&gt;Clarissa, or The History of a Young Lady&lt;/i&gt; is one of the longest novels in the English language. Written by &lt;a href="https://standardebooks.org/ebooks/samuel-richardson"&gt;Samuel Richardson&lt;/a&gt; over a period of several years and published in 1748, it is composed entirely of letters. Though this may seem daunting, the novel is highly regarded and is considered by many critics as one of the greatest works of English literature, appearing in several lists of the best British novels ever written.&lt;/p&gt;
+			&lt;p&gt;The novel tells the story of the young Clarissa Harlowe, eighteen years of age at the start of the novel. She is generally regarded by her family, neighbors, and friends as the most virtuous and kind young woman they know. But she is drawn into correspondence with Richard Lovelace, a well-born, rich young man regarded as something of a rake, when she attempts to reconcile a dispute between Lovelace and her rash brother. Lovelace, imagining this indicates her love for him, carries out a series of strategems which result in him essentially abducting her from her family, from whom Clarissa then becomes estranged.&lt;/p&gt;
```

Comment:
or maybe

```suggestion
			&lt;p&gt;The novel tells the story of a young Clarissa Harlowe, eighteen years of age at the start of the novel. She is generally regarded by her family, neighbors, and friends as the most virtuous and kind young woman they know. But she is drawn into correspondence with Richard Lovelace, a well-born, rich young man regarded as something of a rake, when she attempts to reconcile a dispute between Lovelace and her rash brother. Lovelace, imagining this indicates her love for him, carries out a series of strategems which result in him essentially abducting her from her family, from whom Clarissa then becomes estranged.&lt;/p&gt;
```

We should write her the full name in the lead because it's not in the title anymore.